Zend/zend_builtin_functions.c

```diff
@@ -1625,6 +1625,8 @@ ZEND_FUNCTION(get_defined_vars)
 		return;
 	}
 
+	ZEND_PARSE_PARAMETERS_NONE();
```

Comment:
@nikic Because of this change, the following test failed: https://travis-ci.org/php/php-src/jobs/605445434#L875 . Should I change the expected error message to "get_defined_vars() expects exactly 0 parameters, 2 given" or get rid of this test (because no longer validates what it supposes to) or do something else?

Reply:
You can replace ob_start with register_shutdown_function (and drop the ob_end_clean part).

Reply:
Thanks! I've just pushed the fix.